sc/src/test/java/uk/ac/ebi/atlas/experimentpage/CellMetadataServiceIT.java

```diff
@@ -82,8 +82,8 @@ public void experimentWithMetadataFieldsInIdf() {
     public void experimentWithoutMetadataFieldsInIdf() {
         assertThat(
                 subject.getIdfFileAttributes(
-                        "E-ENAD-13",
-                        jdbcUtils.fetchRandomCellFromExperiment("E-ENAD-13")))
+                        "E-MTAB-5061",
```

Comment:
I would like to move away from using this experiment as it I don't think it has been fully processed. I'm not sure which experiment definitely won't have additional attributes though...